lab3.md

```diff
@@ -324,4 +326,66 @@ In the example we used `sort` from [BusyBox](https://busybox.net/downloads/BusyB
 
 > Tip: did you know that OpenFaaS supports Windows binaries too? Like C#, VB or PowerShell?
 
+### Bonus: Combine all your functions in one yml file
+
+Creating multiple functions means having a `.yml` file for each function.
+
+However, the cli uses `stack.yml` as the default. We can `append` our new functions to the `stack.yml` file and no longer need to use the `-f` flag.
+
+Revisiting the `sorter` function above, if you did rename the astronaut-finder.yml to `stack.yml` you can append the `sorter` to that file and manage both functions with the single yml file.
+
+```bash
+faas-cli new --lang dockerfile sorter --append stack.yml
+```
+
+Your `stack.yml` should now look like this:
+
+```yaml
+provider:
+  name: faas
+  gateway: http://localhost:8080
+
+functions:
+  astronaut-finder:
+    lang: python
+    handler: ./astronaut-finder
+    image: astronaut-finder
+  sorter:
+    lang: dockerfile
+    handler: ./sorter
+    image: sorter
+
+```
+
+Running any of the `faas-cli` commands without the `-f` flag will default to using this `stack.yml` file and will perform the actions on each of the functions listed
+
+```bash
+$ faas-cli build
+> Building astronaut-finder.
+Building: astronaut-finder with python template. Please wait..
+... # build output here
+Image: astronaut-finder built.
+< Building astronaut-finder done.
+> Building sorter.
+Building: sorter with Dockerfile. Please wait..
+... # build output here
+Image: sorter built.
+< Building sorter done.
+worker done.
+```
+
+As you can see, since there was no yml file declared with the `-f` flag, `faas-cli` used the `stack.yml` file as the default, and built both functions that are defined there
+
+You can also use the `--filter` flag to perform actions on specific functions declared in the yml file
+
+```bash
+faas-cli build --filter="sorter"                        
+> Building sorter.
```

Comment:
I am not sure that `--append` works retrospectively.

In that case, we may want to just add another python function in the explanation.